roles/tests-integration/tests/common/mod.rs

```diff
@@ -282,3 +282,28 @@ pub async fn start_template_provider(tp_port: u16) -> TemplateProvider {
     template_provider.generate_blocks(16);
     template_provider
 }
+
+pub async fn start_sv1_mining_device(pool_address: SocketAddr) {
```

Comment:
let's try to be consistent with https://github.com/stratum-mining/stratum/issues/1234

Reply:
also, `pool_address` is a legacy naming convention that's a bit misleading IMO

I would call this `upstream_address`

Reply:
> let's try to be consistent with #1234

No need to return anything here as the mining device is the lowest in the hierarchy,  it never acts as a server